harmonica/gravity_corrections.py

```diff
@@ -69,3 +74,72 @@ def bouguer_correction(topography, density_crust=2670, density_water=1040):
     density[oceans] = -1 * (density_water - density_crust)
     bouguer = 1e5 * 2 * np.pi * GRAVITATIONAL_CONST * density * topography
     return bouguer
+
+def atmospheric_correction(topography):
+    """
+    Calculates atmospheric correction as per Hinze 2005 eqn 5.
+    
+    .. math::
+
+        g_{atm} = 0.874 - 9.9E-5 h + 3.56E-9 h^2 
+
+    in which :math:`h` is the elevation of the station above the ellipsoid
+    and :math:`g_{atm}` is the gravitational effect of the atmospheric the air
+    mass in mGal.
+
+    Parameters
+    ----------
+    topography : array or :class:`xarray.DataArray`
+        Topography height and bathymetry depth in meters.
+        Should be referenced to the ellipsoid (ie, geometric heights).
+        
+
+    Returns
+    -------
+    atmos_correction : array or :class:`xarray.DataArray`
+    The gravitational effect of the atmosphere in mGal
+
+    """
+    atmos_correction = 0.874 - 9.9e-05 * topography + 3.56e-09 * topography**2
+    
+    return atmos_correction
+
+
+def spherical_bouguer_cap_correction(topography):
+    """
+    Calculates spherical cap correction for the Bouguer slab as per Hinze 2013.
+    
+    Gravity and Magnetic Exploration, Cambridge Press 2013 
+    
+    .. math::
+
+        g_{spher} =  0.001464139 h - 3.533047e-07 h^{2} + 1.002709e-13 h^{3}
+        + 3.002407E-18 h^{4} 
```

Comment:
Could we use the analytical formula for the cap instead of this one? That way, the size of the cap can be a parameter instead of restricted to 167km (which might not apply for other planetary bodies or for satellite data).

Reply:
Agreed, see above comment, moving to the spherical bouguer correction allows this.

Reply:
Yes that would be better, you mean as defined by LaFehr 1991 (Geophysics 56)? That allows for density, cap radius and earth radius to be specified.
I've coded this up and compares as follows:
1) Using the approximation (present method)
```
spherical_bouguer_cap_correction(np.array(500))
Out[137]: 0.643743325
```
2) Using the full analytic
```
spherical_bouguer_cap_correction_full(np.array(500))
Out[138]: 0.6440463304229793
```
The approximation assumes a constant density of 2670 kg/m3, so when a different reduction density is used the affect is more apparent. shown here for 2400 kg/m3 density
```
spherical_bouguer_cap_correction_full(np.array(500), 2400)
Out[143]: 0.5789180498184083
```

Reply:
reformulated to include it in the "Bouguer slab formulation" to make it one step
```
 bouguer_correction(np.array(500))
Out[16]: 55.97179595519349
```

The spherical_bouguer_cap_correction now returns both the correction and the spherical corrected infinite slab
```
spherical_bouguer_cap_correction(np.array(500))
Out[17]: (0.6440463304229793, 56.61589317141049)
```

Reply:
Let me know your preference for a 1 step or 2 step correction.

Reply:
Right, I makes more sense now. I'm leaning towards a 1 step process since this is only ever used *with* the Bouguer correction anyway. We could include an argument like `spherical=False` to turn the correction on or off.

Reply:
I agree.  What I proposed is a single function called bouger_correction() where if spherical=True is passed you get the full spherical bouguer correction (the 56.6 mGal value in the above example), otherwise you get the regular infinite slab correction (the 55.9 mGal value).  No need to ever return just the spherical cap correction (0.64 value) as you'd only ever need that to apply to the bouguer_correction.